CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Changelog
 
 ## [Unreleased]
+- Add description tooltips for node and edge types in the weight configuration UI (#1081)
 - Add the `export-graph` command (#1110)
 - Enable loading private repositories (#1085)
 - Enable setting type weights to 0 in the UI (#1005)
 - Add support for 🚀 and 👀 reaction types (#1068)
 - Create one page per project, rather than having a selector (#988)
-<!-- Please add new entries to the _top_ of this section. -->
+  <!-- Please add new entries to the _top_ of this section. -->
 
 ## [0.2.0]
+
```

Comment:
There shouldn't be extra newlines in the older sections of this document

Reply:
Prettier makes these spacing changes to our Changelog file automatically when I save. Including the space beside the `<!--`. Took me a sec to figure out.

Reply:
Ah, I see. I have prettier configured to only run on js files locally. If prettier does this, then it's ok, I try never to argue with automatic code formatters.